transmogrifier/sources/datacite.py

```diff
@@ -137,6 +233,40 @@ def create_from_datacite_xml(
         else:
             kwargs["publication_information"] = [publisher.string]
 
+        # related_items
+        related_items = xml.metadata.find_all("relatedIdentifier")
+        for related_item in related_items:
```

Comment:
It looks like we need to use the `relatedIdentifierType` to construct a URL here like we do for the name identifiers in Creator and Contributor (and probably by calling a very similar function).